recipe_scrapers/addapinch.py

```diff
@@ -9,7 +10,7 @@ def host(cls):
         return "addapinch.com"
 
     def author(self):
-        return self.schema.author()
+        return self.soup.find("meta", {"name": "author"}).get("content")
```

Comment:
It looks like the website has been updated to omit the `|` suffix from the author name field in the `schema.org` metadata, so I don't think this change is necessary (I could be mistaken, let me know if there are other reasons to adjust this).

Reply:
I'm getting a blank response when I run it against the testcase for the grouping. I think it may need to remain as the meta tag unless you see something otherwise!

https://addapinch.com/citrus-scones-with-orange-glaze/